pkg/test/docker.go

```diff
@@ -180,3 +195,46 @@ func (d *dockerRunner) RemoveAllContainers() {
 		}
 	}
 }
+
+func (d *dockerRunner) setHostMapping(resource *dockertest.Resource, mapping hostMapping) error {
+	timeout := time.Duration(1) * time.Millisecond
+	containerIp := resource.Container.NetworkSettings.Networks["bridge"].IPAddress
+	gatewayIp := resource.Container.NetworkSettings.Networks["bridge"].Gateway
+
+	addresses := []string{containerIp, gatewayIp, "127.0.0.1"}
+	for _, ip := range addresses {
+		address := fmt.Sprintf("%s:%d", ip, *mapping.dialPort)
+		if d.isReachable(address, timeout) {
+			d.logger.WithFields(map[string]interface{}{
+				"container": resource.Container.Name[1:],
```

Comment:
why not like this?
```suggestion
				"container": resource.Container.Name,
```

Reply:
Because for some reason `resource.Container.Name` appends a slash to the container name, so I'm just removing it.